sqlglot/expressions.py

```diff
@@ -6427,14 +6427,21 @@ class Base64DecodeBinary(Func):
     arg_types = {"this": True, "alphabet": False}
 
 
+# https://docs.snowflake.com/en/sql-reference/functions/base64_decode_string
 class Base64DecodeString(Func):
     arg_types = {"this": True, "alphabet": False}
 
 
+# https://docs.snowflake.com/en/sql-reference/functions/base64_encode
 class Base64Encode(Func):
     arg_types = {"this": True, "max_line_length": False, "alphabet": False}
 
 
+# https://docs.snowflake.com/en/sql-reference/functions/try_base64_decode_binary
+class TryBase64DecodeBinary(Func):
+    arg_types = {"this": True, "alphabet": True}
```

Comment:
`alphabet` is optional, this is from their docs:

```
TRY_BASE64_DECODE_BINARY(<input> [, <alphabet>])
```

```suggestion
    arg_types = {"this": True, "alphabet": False}
```

Reply:
Ah, yes, that is correct. This is standard in syntax specifications: `[ ... ]` denotes optionality.